nvflare/private/fed/server/server_engine.py

```diff
@@ -67,6 +67,18 @@
 from .server_status import ServerStatus
 
 
+class ClientConnection:
+    def __init__(self, client):
+        self.client = client
+
+    def send(self, data):
+        data = pickle.dumps(data)
+        self.client.send(data)
+
+    def recv(self):
+        return self.client.recv()
```

Comment:
Since we wrap send side using pickle.dumps,
Do we want to have pickle.loads here in the receive side?